document/core/exec/numerics.rst

```diff
@@ -1967,3 +1970,187 @@ Conversions
    \begin{array}{lll@{\qquad}l}
    \narrowu_{M,N}(i) &=& \satu_N(\signed_M(i))
    \end{array}
+
+.. _relaxed-ops:
+
+Relaxed operators
+~~~~~~~~~~~~~~~~~
+
+The result of *relaxed* operators are *host-dependent*, because the set of
+possible results may depend on properties of the host environment (such as
+hardware). Technically, each such operator produces a fixed-size *list of sets*
+of allowed values.  For each execution of the operator in the same environment,
+only values from the set at the same position in the list are returned, i.e.,
+each environment globally chooses a fixed projection for each operator.
+
+.. note::
+   Each operator can be thought of as a family of operations that is fixed to
+   one particular choice by the execution environment. The fixed operation
+   itself can still be non-deterministic or partial.
+
+
+.. _op-frmadd:
+
+:math:`\frmadd_N(z_1, z_2, z_3)`
+................................
+
+Relaxed Multiply Add (madd) allows for fused or unfused results. :math:`fma` is
+defined by |IEEE754|_ (Section 5.4.1) as *fusedMultiplyAdd*.
+
+.. math::
+   \begin{array}{@{}llcll}
+   \EXPROFDET & \frmadd_N(z_1, z_2, z_3) &=& [ \fadd_N(\fmul_N(z_1, z_2), z_3), fma_N(z_1, z_2, z_3) ] \\
+   & \frmadd_N(z_1, z_2, z_3) &=& \fadd_N(\fmul_N(z_1, z_2), z_3) \\
```

Comment:
Do I read this correctly as saying that the deterministic semantics are non-fused?

In [the 01-31 meeting](https://github.com/WebAssembly/meetings/blob/main/main/2023/CG-01-31.md), the vote favored that the deterministic semantics should be fused:

ZA: vote for det semantics for relaxed madd/nmadd:
1. (real) fma
2. mul + add (with intermediate rounding)
3. neutral

**POLL**

|FMA|Mul+Add|Neutral|
|---|--|--|
|15|2|16|

Reply:
This is wrong, needs to be fixed. Thanks for pointing this out.

Reply:
Fixed in [18c665a](https://github.com/WebAssembly/relaxed-simd/pull/115/commits/18c665adbfce7102ba841c01cd61525dd3b6a7b6)